File: KITTY/KURS_1/JAVA/Day_05_Two_App/app/src/main/java/com/example/velic/day_5_two_app/SecondActivity.java
package com.example.velic.day_5_two_app;

import android.os.Bundle;
import android.app.Activity;
import android.widget.TextView;

public class SecondActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        String adress;
        String letter;

        adress = getIntent().getExtras().getString("adress");
        letter = getIntent().getExtras().getString("letter");

        // или переписать так:

        adress = getIntent().getStringExtra("adress");
        letter = getIntent().getStringExtra("letter");


        TextView infoTextView = findViewById(R.id.textViewInfo);
        infoTextView.setText(adress + " , вам передали " + letter);


    }

}

File: KITTY/KURS_1/KOTLIN/DAY_05_1_SecondActivity_kt/app/src/main/java/com/example/velic/day_05_1x_secondactivity/SecondActivity.kt
package com.example.velic.day_05_1x_secondactivity

import android.app.Activity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        val adress = intent.getStringExtra("adress")
        val letter = intent.getStringExtra("letter")
        textViewInfo.text = "$adress , вам передали $letter"
    }

}
